addons/mail/static/src/models/composer_view.js

```diff
@@ -1420,10 +1420,23 @@ registerModel({
                 return this.composer.textInputContent.substring(this.suggestionDelimiterPosition + 1, this.composer.textInputCursorStart);
             },
         }),
+        /**
+         * Reference of the textarea. Useful to set height, selection and content.
+         */
+        textareaRef: attr(),
         textInput: one('ComposerTextInputView', {
             default: {},
             inverse: 'owner',
         }),
+        threadTextualTypingStatusView: one('ThreadTextualTypingStatusView', {
+            compute() {
+                if (this.hasFooter && this.hasThreadTyping) {
```

Comment:
Just `hasThreadTyping`.
`hasFooter` is computed field that depends on `hasThreadTyping`, so useless here